src/main/java/org/opensrp/domain/setting/SettingConfiguration.java

```diff
@@ -29,6 +32,9 @@
 	@JsonProperty
 	private String version;
 	
+	@JsonProperty
+	private String documentId;
```

Comment:
For the other domains the documentId is usually the `id` in the main JSON object, why not maintain that consitency